cmd/flipt/export.go

```diff
@@ -87,6 +88,7 @@ func (c *exportCommand) run(cmd *cobra.Command, _ []string) error {
 		// default to stdout
 		out    io.Writer = os.Stdout
 		logger           = zap.Must(zap.NewDevelopment())
+		enc              = ext.EncodingYML
```

Comment:
should we give the user the ability to set this format via a CLI flag now that we support JSON? could default to yaml still

Reply:
Ahh yeah I wondered about this. So it will support JSON if you specify a file with `.json` as the target.
However, for STDOUT, yeah we could have an actual flag. Same for import and STDIN.